apps/web-tools/pages/sync.tsx

```diff
@@ -0,0 +1,214 @@
+"use client";
```

Comment:
Curious, what is this for?

Reply:
https://nextjs.org/docs/app/building-your-application/rendering/client-components#using-client-components-in-nextjs

I'm not even sure if it is still necessary, but at some point, without it I was getting compiling issues.